From f3a6f62b938815e588bcaa1aefc29fac8b6b137c Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 4 Nov 2016 23:03:12 +0300
Subject: [PATCH 23/24] ARM64: dts: r8a7795-salvator-x: Set Okaya
 RFWSVGAT-7x0AP-APL 1024x600 panel

This sets LVDS LCD parameters for Okaya RFWSVGAT-7x0AP-APL 1024x600 panel.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
index c7497bb..6eb2203 100644
--- a/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77xx-panel.dtsi
@@ -13,20 +13,20 @@
 	lvds: lvds {
 		compatible = "lvds-connector";
 
-		width-mm = <210>;
-		height-mm = <158>;
+		width-mm = <153>;
+		height-mm = <90>;
 
 		panel-timing {
-			/* 1024x768 @60Hz */
-			clock-frequency = <65000000>;
+			/* 1024x600 @60Hz */
+			clock-frequency = <51200000>;
 			hactive = <1024>;
-			vactive = <768>;
-			hsync-len = <136>;
-			hfront-porch = <20>;
-			hback-porch = <160>;
-			vsync-len = <6>;
-			vfront-porch = <3>;
-			vback-porch = <29>;
+			vactive = <600>;
+			hsync-len = <60>;
+			hfront-porch = <160>;
+			hback-porch = <100>;
+			vsync-len = <5>;
+			vfront-porch = <15>;
+			vback-porch = <15>;
 		};
 	};
 };
-- 
1.9.3

